docs/siem/detections/api/exceptions/api-update-exception-item.asciidoc

```diff
@@ -0,0 +1,138 @@
+[[exceptions-api-update-item]]
+=== Update exception item
+
+Updates an existing exception item.
+
+==== Request URL
+
+`PUT <kibana host>:<port>/api/exception_lists/items`
+
+==== Request body
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`comments` |comments[] a|Array of `comment` fields:
+
+* `comment` (string): Comments about the exception item.
```

Comment:
Comments can currently be updated or added, not deleted. I'm working on a PR that adds uuid to comments. So on an update, if comments exist you need to pass in the existing comments as `{ comment: 'some comment', id: '123' }`, any new comments can be appended to the array as `{ comment: "new comment" }`

Reply:
We had to go back on this a bit, and comments are now append only (for the time being). So no editing or deleting of comments 😞

Reply:
The `id` part still stands though, so comment structure is `{ comment: '', id: ''}`